refinery/data_set_manager/tests.py

```diff
@@ -1540,9 +1540,9 @@ def test_update_attribute_order_ranks_invalid(self):
         attribute_list = AttributeOrder.objects.filter(assay=self.assay)
         self.assertItemsEqual(old_attribute_list, attribute_list)
 
-    @mock.patch("data_set_manager.utils.core.utils.get_full_url",
-                return_value="test_file_a.txt")
+    @mock.patch("data_set_manager.utils.core.utils.get_absolute_url")
```

Comment:
Don't see a reason to change the previous format.

Reply:
Format?